misc/test_ping

```diff
@@ -5,31 +5,43 @@ RESULT_LINES=/tmp/result_lines.txt
 MONO_LOG=/tmp/mono-log.txt
 MANIFEST=./module_manifest.json
 REPORT_FILE=/tmp/report.txt
+status=pass
 
 echo Baseline ping test report | tee -a $MONO_LOG
-
 # This is a check that the startup.pcap file exists and is proper.
-echo
-ls -l /scans/startup.pcap
-echo %% $(tcpdump -n -r /scans/startup.pcap | wc -l) packets captured. | tee -a $MONO_LOG
-echo
 tcpdump -en -r /scans/startup.pcap ip
-echo
-tcpdump -ven -r /scans/startup.pcap port 67 | fgrep "length 1: Request"
-echo
-tcpdump -ven -r /scans/startup.pcap port 67 | fgrep "length 1: ACK"
+
+echo %% $(tcpdump -n -r /scans/startup.pcap | wc -l) packets captured. | tee -a $MONO_LOG
 echo
 
+STATIC_IP=`jq -r .static_ip /config/device/module_config.json`
+
+if [ -n "$STATIC_IP" ]; then
+    echo RESULT skip base.startup.request static ip | tee -a $RESULT_LINES
```

Comment:
previous comment still not addressed... ?

https://github.com/faucetsdn/daq/pull/254#discussion_r391899409

Reply:
I missed that one.